src/components/button/Button.js

```diff
@@ -0,0 +1,16 @@
+// components/Button.js
+
+import React from "react";
+import "./Button.css";
+
+function Button({ children, className, onClick }) {
```

Comment:
if you think we will be rendering components inside of our Buttons, then the children prop makes sense (maybe we will want to pass icons?) But if we are really just going to have simple buttons that just have text in them, I think we can skip receiving and rendering children in this component. 
you could just pass buttonText, className, clickHandler and type to the component as props. 
then use it like this:
`<Button buttonText="Submit" className="login-btn" clickHandler={handleLogin} type="submit" />`